app/models/concerns/upstream_only_settings.rb

```diff
@@ -1,6 +1,11 @@
 # Settings to hide in downstream (will return nil for all values)
-module UpstreamOnlySettings
+class UpstreamOnlySettings
   SETTINGS = %w[
-    allow_multiple_content_views
   ].freeze
+
+  def self.include?(key)
+    new.include?
```

Comment:
my bad in the comment
```suggestion
    new.include?(key)
```